src/sgame/sg_main.cpp

```diff
@@ -1135,17 +1135,7 @@ void CalculateRanks()
 
 		if ( level.clients[ clientNum ].pers.connected != CON_DISCONNECTED )
 		{
```

Comment:
Dropping this loop is obviously a good idea.

Reply:
Just cherry-picked them as 55254c94e61609a3fb7aec1ec1bdbe8d7ded5903 and f765dd7966da0c63a76d556ae7997806c8f4ae8e